package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "edit-pixel",
+  "version": "1.0.0",
+  "main": "src/main.js",
+  "repository": "https://github.com/kjleitz/edit-pixel",
+  "author": "tim@timryan.org",
+  "license": "(MIT OR Apache-2.0)",
```

Comment:
I'm not _exactly_ sure what "MIT/Apache-2.0" (from the README) intends, slash-delimited like that, but it seems like `"(MIT OR Apache-2.0)"` probably fits the bill for `package.json`'s `"license"` key, yeah? https://docs.npmjs.com/files/package.json#license